universum/modules/output/html_output.py

```diff
@@ -67,7 +72,8 @@ def log_execution_finish(self, title, version):
         html_footer = "</pre></body></html>"
         self._log_line(html_footer)
 
-    def _log_line(self, line):
+    def _log_line(self, line, with_line_separator=True):
+        line = f"{line}{os.linesep}" if with_line_separator else line
```

Comment:
It looks like the line separator should not be added in case the line already has one.
At least this logic is implemented in the terminal output.
Without it, the html output has the empty line after the "Here is the summarized build result:" string.

Reply:
Fixed.